plugins.json

```diff
@@ -4,7 +4,7 @@
     "author":"MassaLabs",
```

Comment:
@Pierregvx based on the official guide lines, could you please let me know if this "author" info will be displayed on the front-end as it is aka "MassaLabs"? 
If so, we need to change that so it is "Massa Labs" as requested by CEO.
How can we do this? 
cc @pivilartisant @mazmassa it will change the front-end rules I believe (if author = "Massa Labs" then show the "certified" logo)

Reply:
yes it will be displayed how ever the backend sends it, it can be MassaLabs, ML, Massa Labs and all of the above ! 
noted for the FE rukes

Reply:
I changed to Massa Labs
I think we should also change the author in the plugins manifests then
for example [here](https://github.com/massalabs/station-wallet-module/blob/main/manifest.json)

Reply:
yep